src/components/profileEdit/AddressInfo.tsx

```
@@ -137,43 +179,48 @@ export default function AddressInfo() {
                 <h3 className="text-lg font-medium mb-6">배송지 추가</h3>
                     <div className="flex w-full my-3 gap-2 justify-between items-center my-4">
-                        <label className="block text-sm font-medium text-gray-700 mb-1">
-                            주소
-                        </label>
-                        <p className="block text-sm font-medium text-gray-700">{ newAddress.address.length > 0 && `[${newAddress.zipcode}] ${newAddress.address}`}</p>
+                        <div className="flex gap-2 items-center">
+                            <label className="block text-sm font-medium text-gray-700 mb-1 whitespace-nowrap">
+                                주소
+                            </label>
+                            <p className="block text-sm font-medium text-gray-700">{ newAddress.address.length > 0 && `[${newAddress.zipcode}] ${newAddress.address}`}</p>
+                            <InputInfo errorMessage={addressAddError.address} />
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)